chroot-bin/croutoncycle

```diff
@@ -307,6 +328,15 @@ else
         # display will be stuck on the old vt.
         sudo -n chvt 2
         sudo -n chvt "$dest"
+    else
+        dest="/tmp/.X${destdisp#:}-lock"
+        if [ -f "$dest" ]; then
+            # Trigger the target before releasing the current owner
+            kill -USR1 `cat /tmp/.X${destdisp#:}-lock`
+        fi
+        if [ "${freonowner:-0}" != 0 ]; then
+            kill -USR1 "$freonowner"
+        fi
```

Comment:
Yeah, magical flocks o' locks, followed by signal ordering abuse to give Chrome as little time as possible to render anything :)